spk/domoticz/src/dsm-control.sh

```diff
@@ -16,6 +16,14 @@ PORT="8084"
 
 start_daemon ()
 {
+    insmod /lib/modules/usbserial.ko
+    insmod /lib/modules/ftdi_sio.ko
+    insmod ${INSTALL_DIR}/modules/cp210x.ko
```

Comment:
There's a lot of comments over in the HDHomeRun/TVHeadEnd threads about this kind of issue.
I know with LIRC I did a full rmmod of anything we inserted, but that is not necessarily the best idea, as already mentioned above (shared usb modules etc).

The whole "package" infrastructure makes it a bit tough because ideally things should be completely independent and removing everything from a package installation shouldn't impact anything else on the system - but we have well and truly closed that line! Ideally rather than directly loading and unloading, we could have a system in place where we could package-specific module requirements into config files (e.g. /var/packages/spksrc_drivers/modules.d/...) with a separate package that handles loads/unloads and dependencies as required; however we don't have such a process in place, and that would still only help with our (spksrc) packages and there are many other package providers who wouldn't comply.... can't win!

Best practice here would probably be to unload package-specific modules (cp210x) on exit and leave potentially shared modules loaded. Agree?